java-security/src/main/java/com/sap/cloud/security/token/x509/X509CertSelector.java

```diff
@@ -0,0 +1,11 @@
+package com.sap.cloud.security.token.x509;
```

Comment:
this schould be moved to java-api.

Reply:
as discussed not moving due to create() method